packages/manager/apps/pci-users/src/pages/OnBoardingPage.tsx

```diff
@@ -0,0 +1,118 @@
+import { useNavigation } from '@ovh-ux/manager-react-shell-client';
+import { Card, OnboardingLayout } from '@ovhcloud/manager-components';
+import { OdsBreadcrumbAttributeItem } from '@ovhcloud/ods-components';
+import { OsdsBreadcrumb } from '@ovhcloud/ods-components/react';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Outlet,
+  useHref,
+  useParams,
+  useRouteLoaderData,
+} from 'react-router-dom';
+import { Project } from '@/data/project';
+
+const OnBoardingPage: React.FC = () => {
```

Comment:
React.FC is not the preferable way to type a react component, use JSX.Element instead, it's what'is used through the whole project

Reply:
Update that and it's ok for me

Reply:
can i know what's the reason behind and what make the React.FC not good practice to use ?

Reply:
https://github.com/facebook/create-react-app/pull/8177

Reply:
Convinced, thank u